besser/bot/nlp/nlp_engine.py

```diff
@@ -61,6 +62,8 @@ def initialize(self) -> None:
         self._ner = SimpleNER(self, self._bot)
         if self.get_property(nlp.NLP_STT_HF_MODEL):
             self._speech2text = HFSpeech2Text(self)
+        if self.get_property(nlp.NLP_STT_SR_ENGINE):
```

Comment:
I would add an 'else' here, both properties are not supposed to be set at the same time, but it makes more sense to me